docker-database/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+services:
+  db:
+    container_name: database
+    build: DB_dockerDir/
```

Comment:
Personal preference : in such case, i would avoid building a custom container and bind mount the sql and csv files from docker compose instead to have a faster lifecycle when editing it. But what you do is more portable so its more a matter of taste

Reply:
Wise advise. I will leave it like that as I might have to load other scripts. I will revisit it when the prototype is more mature.